support/docker/dev/janitor.json

```diff
@@ -19,13 +19,17 @@
       "proxy": "https"
     },
     "9000": {
-      "label": "PeerTube",
+      "label": "PeerTube API",
+      "proxy": "https"
+    },
+    "3000": {
+      "label": "PeerTube front",
```

Comment:
Nit: What about "web app" instead of "front"? I'd find that more explicit/transparent.

Reply:
If I wanted to be picky, I'd say the API also uses web technologies and is an app, so it's also a web app ;) But yeah, it probably makes more sense overall :)